Druid/DRBytecodeGenerator.class.st

```diff
@@ -250,12 +261,19 @@ DRBytecodeGenerator >> visitReturn: aDRReturn [
 ]
 
 { #category : 'ir-to-target' }
-DRBytecodeGenerator >> visitStoreInstVar: aDRStoreInstanceVariable [ 
-
+DRBytecodeGenerator >> visitStoreInstVar: aDRStoreInstanceVariable [
+
+	(self isSelf: aDRStoreInstanceVariable) ifTrue: [
```

Comment:
Here I would expect `self isSelf: aDRStoreInstanceVariable operand1` (the store is not self, neither the load). It's more clear.

Also, it's coincidence that both (load and store) have the receiver object as `operand1`. It has no too much sense to have it in `isSelf:`.